pkg/drivers/utils/utils.go

```diff
@@ -1088,3 +1090,20 @@ func GetOcpNsUidGid(nsName string, psaJobUid string, psaJobGid string) (string,
 	}
 	return psaJobUid, psaJobGid, isOcp, nil
 }
+
+// PauseCleanupResource returns whether to pause the cleanup the CRs & other resources.
+func PauseCleanupResource() (time.Duration, error) {
+	pauseCleanupVal := time.Duration(0)
+	pausecleanupValStr, err := k8sutils.GetConfigValue(KdmpConfig, defaultPXNamespace, PauseResourceCleanupKey)
```

Comment:
```suggestion
	pauseCleanupValStr, err := k8sutils.GetConfigValue(KdmpConfig, defaultPXNamespace, PauseResourceCleanupKey)
```
Let's keep the naming consistent